assets/javascripts/discourse/controllers/subscriptions.js

```diff
@@ -0,0 +1,37 @@
+import Controller from "@ember/controller";
+import Ember from "ember";
+import { htmlSafe } from "@ember/template";
+import I18n from "I18n";
+
+export default Controller.extend({
```

Comment:
We no longer prefer this way of defining controllers or components in Ember. 
We instead use native classes.

`export default class SubscriptionsController extends Controller {`